.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
```

Comment:
definitely don't gitignore package.json or package-lock.json as this will render you repo unusable to anyone except you